test/units/TEST-74-AUX-UTILS.keyutil.sh

```diff
@@ -47,4 +47,11 @@ testcase_public() {
     (! /usr/lib/systemd/systemd-keyutil public)
 }
 
+testcase_pkcs7() {
+    echo -n "test" > /tmp/payload
+    openssl dgst -sha256 -sign /tmp/test.key -out /tmp/payload.sig /tmp/payload
+    /usr/lib/systemd/systemd-keyutil --certificate /tmp/test.crt --output /tmp/payload.p7s pkcs1-to-pkcs7 /tmp/payload.sig
+    openssl smime -verify -binary -inform der -in /tmp/payload.p7s -content /tmp/payload -certfile /tmp/test.crt -nointern -noverify > /dev/null
```

Comment:
`-verify` and `noverify`?

Reply:
The second one is about self-signing vs CA:

```
40F7E8CB487F0000:error:10800075:PKCS7 routines:PKCS7_verify:certificate verify error:../crypto/pkcs7/pk7_smime.c:301:Verify error: self-signed certificate
```